packages/cli/src/commands/logout.ts

```diff
@@ -1,6 +1,5 @@
 import chalk from 'chalk';
 import logo from '../util/output/logo';
-// @ts-ignore
```

Comment:
Apparently not necessary 🤷

Reply:
Right, the comment `@ts-ignore` is intended to ignore TS errors (**not recommend**), If what I code is well, not necessary that comment : -)